### ISSUE TITLE
Use time calculation without print or publish

### ISSUE THREAD

**Issue by adamjohansson** [equipe/equipe_issues#80]
Possiblilty to use the time, but don't show it on online/print outs/scoreboards

**Comment by frofrik** [comment]
Detta hör ihop med #54 tänker jag

**Comment by jnstq** [comment]
Nej, detta är att kunna använda tid, men att vi inte publicerar den, nu när vi tar bort tiden så finns den inte att mata in, men man ska kunna välja att dölja den men ändå ta tid.
